hw/chisel/src/main/scala/snax/xdma/xdmaExtension/VerilogMemset.scala

```diff
@@ -0,0 +1,21 @@
+package snax.xdma.xdmaExtension
+
+import chisel3._
+import chisel3.util._
+import snax.xdma.DesignParams.DMAExtensionParam
+
+object HasVerilogMemset extends HasDMAExtension {
+  implicit val extensionParam: DMAExtensionParam = new DMAExtensionParam(
+    moduleName = "VerilogMemset",
+    userCsrNum = 1,
+    dataWidth = 512
+  )
+  def instantiate(clusterName: String): SystemVerilogDMAExtension = Module(
+    new SystemVerilogDMAExtension(
+      topmodule = "VerilogMemset",
+      filelist = Seq("src/main/systemverilog/VerilogMemset/VerilogMemset.sv")
```

Comment:
This means anyone who adds a new extension needs to be in the `src/main/systemverilog/` directory?

So say for example if I have an HDC accelerator Iw ould put `src/main/systemverilog/HdcAccelerator/HdcAccelerator.sv`? Or I have the option to modify this?

Also wait, so if I put a new extension, I need to put a new `.scala` file like `HdcAccelerator.scala` like this? Did I understand it correctly?

Reply:
For this generator, the actual directory of SystemVerilog is not important, as all the content of SystemVerilog file will be copied to the output file. However, it is important to provide Chisel with the correct file list.